tests/unit/conftest.py

```diff
@@ -28,6 +29,8 @@
 
 test_dbx_config = DatabricksConfig.from_token(TEST_HOST, TEST_TOKEN)
 
+rich.reconfigure(soft_wrap=True)
```

Comment:
hey @copdips , 
you can remove this line, I've added it in 0.7.6. 

Please also mention the fix in the CHANGELOG and it's good to merge.

Reply:
thx and fixed